src/transformers/pipelines/image_to_text.py

```diff
@@ -78,29 +84,43 @@ def __call__(self, images: Union[str, List[str], "Image.Image", List["Image.Imag
         Args:
             images (`str`, `List[str]`, `PIL.Image` or `List[PIL.Image]`):
                 The pipeline handles three types of images:
-
                 - A string containing a HTTP(s) link pointing to an image
                 - A string containing a local path to an image
                 - An image loaded in PIL directly
-
                 The pipeline accepts either a single image or a batch of images.
-
             max_new_tokens (`int`, *optional*):
                 The amount of maximum tokens to generate. By default it will use `generate` default.
-
             generate_kwargs (`Dict`, *optional*):
                 Pass it to send all of these arguments directly to `generate` allowing full control of this function.
-
         Return:
             A list or a list of list of `dict`: Each result comes as a dictionary with the following key:
-
             - **generated_text** (`str`) -- The generated text.
         """
         return super().__call__(images, **kwargs)
 
-    def preprocess(self, image):
-        image = load_image(image)
-        model_inputs = self.image_processor(images=image, return_tensors=self.framework)
+    def preprocess(self, images, preprompt=None):
```

Comment:
```suggestion
    def preprocess(self, images, prompt=None):
```
Any reason this is called preprompt instead of prompt?